lib/chai-files.js

```diff
@@ -0,0 +1,108 @@
+var fs = require('fs');
```

Comment:
it should definitely live in it's own module. I just used this place to for a first draft that can be extracted out later. the error messages can also still be improved.